roles/deploy_olm_operator/tasks/main.yml

```diff
@@ -233,6 +233,19 @@
     - name: "Failed on deploy_olm_operator_results"
       fail:
         msg: "{{ deploy_olm_operator_results }}"
+    
+    - name: "Get replicaset logs for operators that failed olm-deployment output"
+      shell: "{{ oc_bin_path }} get replicasets -o go-template={%raw%}'{{range .items}}{{.status.conditions}}{{"\n"}}{{end}}'{%endraw%} -n {{ openshift_namespace }}"
+      register: olm_deployment_output
+      environment:
+        KUBECONFIG: "{{ kubeconfig_path }}"
+      when: deploy_olm_operator_results.result == 'fail'
+    
+    - name: "Output logs for operators that failed olm-deployment output to a debug file"
+      copy:
+        content: "{{ olm_deployment_output.stdout }}"
+        dest: "{{ work_dir }}/operator-olm-deployment-output.txt"
```

Comment:
Like above, renaming the resulting file to `operator-olm-replicaset-output.txt` would make more sense.

Reply:
Ok, this filename was mentioned in deliverables in Jira task, changing it.